warehouse/config.py

```diff
@@ -358,6 +368,10 @@ def configure(settings=None):
             ],
         )
 
+        # For development only: this artificially prolongs the expirations of any
+        # Warehouse-generated TUF metadata by approximately one year.
+        settings.setdefault("tuf.development_metadata_expiry", 31536000)
```

Comment:
This doesn't seem to be used anywhere? Also, if we want to override this for development, we should set a default value to be used in production, and then override it in the `dev/environment` file.